### PR TITLE
Feature/aos 3026 multiple containers

### PR DESCRIPTION
**Pull request by bjartek**
Resolve the dockerTagReference to the Digest-Content-Sha header that is in the registry and return an isLatest property to see if it matches the running sha or not.

Add a list of containers to podResource with detailed information about each continer. 

Keep aggregated fields so that it will work with old konsoll.

Boy Scout:

- refactor dataloader loading to use the class name of the dataloader
- structure dataloader and resolver in same file
- use v6 pipeline
- log if you overwrite the docker registry
- remove configuration for docker registry from main profile

### CODE REVIEW

**Comment by bjartek** [comment]
Testing graphql with webTestClient is done in lots of different ways. Could we for tests that test a lot of assert simply assert that is equal to a file in the resource folder?

File: src/main/kotlin/no/skatteetaten/aurora/gobo/integration/imageregistry/ImageRegistryServiceBlocking.kt [hunk 1]
```diff
@@ -26,6 +31,28 @@ class ImageRegistryServiceBlocking(
 
     private val objectMapper = jacksonObjectMapper()
 
+    fun resolveTagToSh(key: ImageTag): String {
```
Comment (jarlehansen) [inline src/main/kotlin/no/skatteetaten/aurora/gobo/integration/imageregistry/ImageRegistryServiceBlocking.kt]: resolveTagToSh missing an "a" at the end?

File: src/main/kotlin/no/skatteetaten/aurora/gobo/integration/imageregistry/ImageRegistryServiceBlocking.kt [hunk 2]
```diff
@@ -26,6 +31,28 @@ class ImageRegistryServiceBlocking(
 
     private val objectMapper = jacksonObjectMapper()
 
+    fun resolveTagToSha(key: ImageTag): String {
```
Comment (jarlehansen) [inline src/main/kotlin/no/skatteetaten/aurora/gobo/integration/imageregistry/ImageRegistryServiceBlocking.kt]: Probably should have a test?

Comment (bjartek) [reply src/main/kotlin/no/skatteetaten/aurora/gobo/integration/imageregistry/ImageRegistryServiceBlocking.kt]: ApplicationWithLatestDigestQueryResolverTest ?

**Pull request merged**